toolchest_client/tools/star.py

```diff
@@ -30,9 +30,11 @@ def __init__(self, tool_args, output_name, inputs, input_prefix_mapping,
             max_inputs=2,
             database_name=database_name,
             database_version=database_version,
-            parallel_enabled=True,
+            parallel_enabled=True if parallelize else False,
             max_input_bytes_per_file=128 * 1024 * 1024 * 1024,
-            output_type=OutputType.SAM_FILE,  # In many cases, this will change (e.g. a dangerous argument is passed)
+            max_input_bytes_per_file_parallel=4.5 * 1024 * 1024 * 1024,
+            output_type=OutputType.SAM_FILE if parallelize else OutputType.GZ_TAR,
+            output_is_directory=False if parallelize else True,
```

Comment:
^, `output_is_directory=not parallelize`?

Reply:
Updated

Reply:
(different than the above from my perspective, as this includes an implicit boolean cast)